ambry-api/src/main/java/com/github/ambry/config/ReplicationConfig.java

```diff
@@ -265,8 +265,38 @@ public class ReplicationConfig {
   @Config(REPLICATION_THREAD_TYPE)
   public final String replicationThreadType;
 
+  /**
+   * Config for file manager class for backup checker
+   */
+  public static final String BACKUP_CHECKER_FILE_MANAGER_TYPE = "backup.checker.file.manager.type";
+  public static final String DEFAULT_BACKUP_CHECKER_FILE_MANAGER = "com.github.ambry.replication.BackupCheckerFileManager";
+  @Config(BACKUP_CHECKER_FILE_MANAGER_TYPE)
+  public final String backupCheckFileManagerType;
+
+  /**
+   * Config for folder to store all results from backup checker
+   */
+  public static final String BACKUP_CHECKER_REPORT_DIR = "backup.checker.report.dir";
+  public static final String DEFAULT_BACKUP_CHECKER_REPORT_DIR = "/tmp";
```

Comment:
Not sure if /tmp is writable on the server. But we can change the config any way.

Reply:
this is just for open source, on the server the config will be different something like /export/lid etc.